nex/runner.go

```diff
@@ -55,7 +56,7 @@ func RunWorkload(ctx context.Context, logger *slog.Logger) error {
 		return err
 	}
 
-	nodeClient := controlapi.NewApiClientWithNamespace(nc, Opts.Timeout, Opts.Namespace, logger)
+	nodeClient := controlapi.NewApiClientWithNamespace(nc, time.Millisecond*25000, Opts.Namespace, logger)
```

Comment:
Was it intentional to change this from an option value to a hard-coded value? Does this make the option not useable anymore? And again, what led to determining this specific timeout?

Reply:
In order to make the run requests behave synchronously, I changed this to `25000` milliseconds to match the maximum amount of time `MachineManager` would wait for an agent to be available (which is now 5 attempts * 5000ms).

Reply:
> (which is now 5 attempts * 5000ms).

Actually, this is only true now as a result of [this](https://github.com/synadia-io/nex/pull/378/commits/3355cfd8993692ec35c4233222e5e11b602c9c6d) commit.